src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("The Figure: Circle; radius = " + radius
+                + "\n" + " Area: " + getArea()
+                + "; Color: " + getColor() + "\n");
+    }
```

Comment:
Please, pay attention to the example result in task description. For all figures